databuilder/query_language.py

```diff
@@ -316,6 +316,12 @@ def to_first_of_year(self):
     def to_first_of_month(self):
         return _apply(qm.Function.ToFirstOfMonth, self)
 
+    def is_between(self, begin, end):
+        return self.__gt__(begin) & self.__lt__(end)
```

Comment:
Mixing methods and operators like this feels a bit inconsistent to me. I think it either wants to be:
```py
return (self > begin) & (self < end)
```
or
```py
return self.__gt__(begin).__and__(self.__lt__(end))
```

I think my preference would be for the former as it's a lot more readable, although there's an argument that using operators within the definition of operator methods might be mixing layers a bit.

Reply:
Good point, done